src/Stratis.Bitcoin.Features.PoA/Voting/DefaultVotingController.cs

```diff
@@ -35,6 +36,30 @@ public DefaultVotingController(IFederationManager fedManager, ILoggerFactory log
             this.logger = loggerFactory.CreateLogger(this.GetType().FullName);
         }
 
+        /// <summary>
+        /// For simplicity have put this here. It's currently the only place to get information about the current mining key on CirrusMinerD.
+        /// </summary>
+        [Route("info")]
+        [HttpGet]
+        public IActionResult GetInfo()
```

Comment:
There is already an endpoint that does that: `/api/DefaultVoting/fedmembers`

It returns: 

```
[
  "PubKey:'03025fcadedd28b12665de0542c8096f4cd5af8e01791a4d057f67e2866ca66ba7'",
  "PubKey:'027724a9ecc54417ff0250c3355d300cee008747b630f43e791cd02c2b35294d2f'",
  "PubKey:'022f8ad1799fd281fc9519814d20a407ed120ba84ec24cca8e869b811e6f6d4590'"
]
```